static/js/ajax-notifications.js

```diff
@@ -26,13 +26,13 @@ function displayPNotify(type, title, message) {
     });
 }
 
-function WSCheckNotifications() {
+function wsCheckNotifications() {
     var ws = new WebSocket('ws://' + window.location.hostname + ':' + window.location.port + '/ws' + WSMessageUrl);
-    ws.onmessage = function (evt) {
+    ws.onmessage = function(evt) {
         var msg;
         try {
             msg = JSON.parse(evt.data);
-        } catch(e) {
+        } catch (err) {
```

Comment:
No and this isn't `err` this is `e`. err should return a new error object where as this is an exception object. There's not actually any errors as this is used to check if the string if a json encoded string or just a normal string.

If xo is giving an error just add this to the right of it.
`// eslint-disable-line-only ADD_ERROR_NAME_HERE`